docs/updating-ccid.md

```diff
@@ -0,0 +1,77 @@
+# Updating the CCID free software driver
+
+Note: This page is mainly intended for the Chrome OS Smart Card Connector
+project maintainers.
+
+## Background
+
+The Chrome OS Smart Card Connector app is shipped with a specific version of the
+CCID free software driver. This is a driver for smart card reader (not to be
+confused with drivers for smart cards, also sometimes called middleware).
+
+Reasons for pinning to a specific version of the driver:
+
+* potentially, adjustments in our code or in the building scripts might be
+  required after updating to new versions (note that we don't run the driver
+  verbatim, but instead port it under the Native Client technology for running
+  inside the JavaScript-based App);
+* we can make sure that the driver that we roll out as part of the App underwent
+  some QA testing.
+
+However, this approach also implies some amount of maintenance work for updating
+the driver (say, every few months). It also means that the users typically get
+a somewhat outdated version of the driver.
+
+## Update procedure
+
+0. The currently used version of the driver is located under the
+   `//third_party/ccid/` directory. Specifically, the
+   `//third_party/ccid/README.google` file should contain the version number,
```

Comment:
nit: 'contains' instead of 'should contain'. Or, in the line below, replace 'contains' with 'should contain' (I would favor 'contains' in both places, but more important is consistency).

Reply:
Done. (I hope we won't forget to update the readme file :) )